src/tests/10_TrendingComapnies.test.ts

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai'
+import TrendingCompaniesService from '../db/service/trendingCompaniesService'
+
+describe.only('Trending Companies service can', () => {
+    let trendingCompaniesService: TrendingCompaniesService
+	it('be setup', async () => {
+        trendingCompaniesService = new TrendingCompaniesService();
+	})
+
+    it('can successfully pull data from the API',async()=>{
+        const response = await trendingCompaniesService.getTrendingCompanies();
+        console.log(response.message.length);
```

Comment:
Since we want the top 10, we should have this test check if the length of the array is ten.

```suggestion
        expect(response.trendingCompanies.length).equals(10)
```